src/cartridges/int_adyen_overlay/cartridge/static/default/css/adyenCss.css

```diff
@@ -420,6 +420,43 @@ input:checked + .slider:before {
     width: 100%;
     padding: 10px 0;
 }
+
+.express-loader-container {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.express-loader {
+    width: 48px;
+    height: 48px;
+    border: 5px solid #00a1e0;
+    border-bottom-color: transparent;
+    border-radius: 50%;
+    display: inline-block;
+    box-sizing: border-box;
+    animation: rotation 1s linear infinite;
+}
+
+.hidden {
+    visibility: hidden;
+    height: 0;
+}
+
+.adyen-checkout__amazonpay__button {
+    width: 100% !important;
```

Comment:
can we do it without the !important? what if we try min/max width, or inline styling?

Reply:
yeah, min-width works 👍